.github/workflows/openconcept.yaml

```diff
@@ -0,0 +1,51 @@
+name: Build
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  Linux_Build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set Python Version
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
+    - name: Install Dependencies
+      run: |
+        pip install wheel
+        pip install -r requirements.txt
+    - name: Build and Test
+      run: python -m pytest --cov-config .coveragerc --cov=openconcept
```

Comment:
Any idea why we don't `pip install` the project for Linux/Windows? Just install dependencies and test? Are we relying on the fact that the current directory is on the PYTHONPATH? IMO that's super sketchy.

Reply:
I will default to @bbrelje for this. I copied the workflow used in Travis, but we can change it if Ben does not have an issue with the modification.

Reply:
@nwu63 The reason is that Anaconda is able to get the high-performance Intel numpy/scipy and pip is not, and I want that incorporated in the Windows build / test

Reply:
I see, yeah for the Windows build we can just run `conda install numpy scipy` (with the correct versions) before running `pip install .`, so that `conda` would handle the installation for those. Btw I don't think `conda` would install the intel distribution by default, that requires running ` conda install -c intel numpy`. If you want we can run that on Windows instead of the standard `numpy` provided by `conda`.

Also, are you okay with running the Windows builds on GA instead of AppVeyor? If so then after this is merged I might ping you to disable the build trigger on there.

Reply:
Using `condo install -c intel numpy` did not work and resulted in errors at runtime (not during the installation). If that is an important component of the testing, I will debug. If not, the standard installation of numpy is working.